seacrowd/sea_datasets/ud/ud.py

```diff
@@ -0,0 +1,394 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+from conllu import TokenList
+
+from seacrowd.utils import schemas
+from seacrowd.utils.common_parser import load_ud_data, load_ud_data_as_seacrowd_kb
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks
+
+
+_CITATION = ""
+
+_LANGUAGES = ["ind", "vie"]
```

Comment:
Could you please include "tgl" too in this dataloader?

This dataset also has Tagalog subsets on HuggingFace: `tl_trg` and `tl_ugnayan`.
Thanks for letting us know, @tellarin!

Reply:
Hi @holylovenia @tellarin , thanks for pointing this out. I've checked all of these subsets (incl in your latest recommendation to be added) are available on the [UD list on homepage](https://github.com/UniversalDependencies).